app/components-react/highlighter/SettingsView.tsx

```diff
@@ -155,18 +156,32 @@ export default function SettingsView({
                   )}
                 </p>
 
-                <SwitchInput
-                  style={{ margin: 0, marginLeft: '-10px' }}
-                  size="default"
-                  value={v.useAiHighlighter}
-                  onChange={toggleUseAiHighlighter}
-                />
+                {v.highlighterVersion !== '' ? (
+                  <SwitchInput
+                    style={{ margin: 0, marginLeft: '-10px' }}
+                    size="default"
+                    value={v.useAiHighlighter}
+                    onChange={toggleUseAiHighlighter}
+                  />
+                ) : (
+                  <Button
+                    style={{ width: 'fit-content' }}
+                    type="primary"
+                    onClick={() => {
+                      HighlighterService.installAiHighlighter(true);
```

Comment:
not blocking but we probably want to use `HighlighterService.actions.installAiHighlighter(true)` here-- the action dispatcher lets it process asynchronously and helps performance